### PR TITLE
Add listing of how many hours each user is dutying

### LINKED ISSUE #73: Add listing of how many hours each user is dutying

### PR DESCRIPTION
**Pull request by lye-jw**
Fixes #73 
Shows current user's total hours on home page

### CODE REVIEW

**Comment by indocomsoft** [comment]
@lye-jw I think the purpose of this is more to show how many hours someone has dutied in a particular week because there’s a limit of 16 hours. This is connected to #74

**Comment by lye-jw** [comment]
How are 30-min slots factored into total duty hours by the university?

**Comment by lye-jw** [comment]
Displays current user's hours at homepage. Displays in red if exceeds 16 hours.

File: app/views/duties/index.html.erb [hunk 1]
```diff
@@ -25,6 +25,11 @@
         <% if can?(:update_default_duties, Duty) %>
           <%= link_to 'Update Default Duties', availabilities_places_path, class: "btn btn-large btn-light duty-table-button" %>
         <% end %>
+        <% if current_user_hours > 16 %>
+          <div class = "duty-hours-exceed">Your hours: <%= current_user_hours %></div>
+        <% else %>
+          <div class = "duty-hours">Your hours: <%= current_user_hours %></div>
+        <% end %>
```
Comment (indocomsoft) [inline app/views/duties/index.html.erb]:
```ruby
<div class = "duty-hours<%= current_user_hours > 16 ? " duty-hours-exceed" : "" %>">
  Your hours: <%= current_user_hours %>
</div>
```

File: app/views/duties/index.html.erb [hunk 2]
```diff
@@ -25,6 +25,9 @@
         <% if can?(:update_default_duties, Duty) %>
           <%= link_to 'Update Default Duties', availabilities_places_path, class: "btn btn-large btn-light duty-table-button" %>
         <% end %>
+        <div class = "duty-hours <%= current_user_hours > 16 ? "duty-hours-exceed" : "" %>" >
```
Comment (indocomsoft) [inline app/views/duties/index.html.erb]:
```suggestion
        <div class = "duty-hours<%= current_user_hours > 16 ? " duty-hours-exceed" : "" %>" >
```

Shouldn’t have a space if there’s only 1 class here.

Comment (indocomsoft) [reply app/views/duties/index.html.erb]: But otherwise everything LGTM! :)

**Pull request merged**